index.bs

```diff
@@ -1073,19 +1073,19 @@ Methods</h4>
 			1. Let <var>promise</var> be a new promise.
 
 			2. If the operation <a href="https://tc39.github.io/ecma262/#sec-isdetachedbuffer"><code>IsDetachedBuffer</code></a>
-				(described in [[!ECMASCRIPT]]) on <a>audioData</a> is
+				(described in [[!ECMASCRIPT]]) on {{BaseAudioContext/decodeAudioData()/audioData!!argument}} is
```

Comment:
Not sure why I need `BaseAudioContext/decodeAudioData()/` here.  `audioData` is unique and only occurs in the `decodeAudioData` method.  bikeshed complains without this.

Reply:
That is a bug on my part, and it's probably present in a number of places. 

The argumentdef block doesn't properly handle arg-less methods in its `for` attribute, but it *should* as long as the argless form is unambiguous (there's only one overload of the method). It's something on my list to fix, don't worry about these errors for now, as they'll go away when I do.

(The end result of this is that you accidentally end up with two versions of the argument being defined sometimes; you can see this with `bikeshed refs --text=audioData --type=argument`.)

Reply:
So I should undo the `BaseAudioContext/decodeAudioData()/` additionsk, and just leave it as `{{audioData!!argument}}`?  WFM, if so.

Reply:
Yeah. Or just leave it as {{audioData}}, if that doesn't conflict with something else.

Reply:
`{{audioData}}` works fine and all links point to the correct place.